app/views/candidate_mailer/pool_opt_in.text.erb

```diff
@@ -0,0 +1,41 @@
+Dear <%= @application_form.first_name %>,
+
+You have chosen to share your application details with training providers who have spaces on their courses. [Read more about how application sharing works](<%= @share_details_url %>).
+
+[Change your decision or update your preferences](<%= @update_preferences_url %>)
+
+## What is application sharing?
+When you have no other applications in progress, your details will be visible in a list that training providers can search through.
+
+They can see your:
+* qualifications
+* work history and unpaid experience
+* most recent personal statement
+* basic information about where you have applied to before, including the name of the training provider and the subject you applied for
+
+If you meet the criteria for courses that they have spaces on, they can send you an invitation to submit an application.
+
+You can decide whether you want to submit an application or not. If you do, then the provider will treat it the same as other applications they receive and decide whether to offer you an interview or not.
+
+^ Do not wait to be invited to apply. Providers offer places on their courses throughout the year and there is a risk that you will not receive any offers if you wait to be invited to apply.
+
+## Will courses I’m invited to be relevant?
+Training providers will filter the list of candidates by:
+* the subjects you have applied for in the past
+* whether you are interested in full or part-time courses
+* whether you would apply for fee-paying or salaried courses only
+* the location that you have said you can train in
+
+This means you should only get invitations to courses that meet your needs. You can [update your preferences](<%= @update_preferences_url %>) to receive more relevant invitations.
+
+## When am I visible in the list?
+Training providers cannot invite candidates who have applications in progress with another training provider. This would give some training providers an unfair advantage in recruitment.
+
+So that the courses you have applied to can have time to review your application properly, we will hide you from other training providers until they have made a decision.
+
+This means that you will only be available for other training providers to invite you to apply to their courses when all of your applications are either:
+* withdrawn by either you or the training provider
+* rejected
+* inactive - which means that the training provider has taken more than 30 working days to make a decision.
+
+You can change your mind about sharing your details or update your preferences in the [application sharing tab in Apply for teacher training courses](<%=  @application_sharing_url %>).
```

Comment:
If this is not considered a transactional email, we'll need to add the unsubscribe link here, rendering the partial `app/views/candidate_mailer/_unsubscribe_from_emails_like_this.text.erb`